cmd/fetch.go

```diff
@@ -37,8 +37,8 @@ func fetchCmd() *cobra.Command {
 	fetchCmd := &cobra.Command{
 		Use:   "fetch [flags] SBOM_URL...",
 		Args:  cobra.MinimumNArgs(fetchMinArgs),
-		Short: "Fetch SBOM file(s) from HTTP(S), OCI, or Git URLs",
-		Long:  "Fetch SBOM file(s) from HTTP(S), OCI, or Git URLs",
+		Short: "Fetch SBOM file(s) from HTTP(S), OCI, Git or Github URLs",
+		Long:  "Fetch SBOM file(s) from HTTP(S), OCI, Git, or Github URLs",
```

Comment:
```suggestion
		Short: "Fetch SBOM file(s) from HTTP(S), OCI, Git, or Github URLs",
		Long:  "Fetch SBOM file(s) from HTTP(S), OCI, Git, or Github URLs",
```
Nit: Not sure if the intention was for them to be the same